lbz/resource.py

```diff
@@ -89,7 +90,11 @@ def __repr__(self) -> str:
         return f"<Resource {self.method} @ {self.urn} >"
 
     def _load_configuration(self) -> None:
-        self.auth_enabled = env.get("ALLOWED_PUBLIC_KEYS") or env.get("ALLOWED_AUDIENCES")
+        try:
+            PUBLIC_KEYS.value  # pylint: disable=pointless-statement
+            self.auth_enabled = True
+        except MissingConfigValue:
```

Comment:
We should never base on this Exception, so, is `ALLOWED_PUBLIC_KEYS` required or not?